taichi/ui/backends/vulkan/renderer.h

```diff
@@ -78,7 +78,7 @@ class TI_DLL_EXPORT Renderer {
   taichi::lang::StreamSemaphore get_render_complete_semaphore();
 
  private:
-  glm::vec3 background_color_ = glm::vec3(0.f, 0.f, 0.f);
+  glm::vec3 background_color_ = glm::vec3(0.f, 0.f, 0.0f);
```

Comment:
why is this needed?

Reply:
Sorry, I want to change `background_color_` to a default color, but I found that some tests without calling `clear_color` failed to pass. So later I cancled this feature. Just handwriting fault.